what4/src/What4/Protocol/SMTLib2.hs

```diff
@@ -1150,8 +1151,8 @@ shutdownSolver _solver p = do
 -----------------------------------------------------------------
 -- Checking solver version bounds
 
-mkChunks :: [Word] -> [Versions.VChunk]
-mkChunks = map ((:[]) . Versions.Digits)
+mkChunks :: Word -> [Word] -> NonEmpty Versions.VChunk
```

Comment:
I like this, since it's likely the major version number that will be most significant in determining compatibility.